client/lib/ffi.rs

```diff
@@ -35,23 +35,24 @@ pub enum casper_error_t {
     CASPER_FAILED_TO_PARSE_TIME_DIFF = -4,
     CASPER_FAILED_TO_PARSE_TIMESTAMP = -5,
     CASPER_FAILED_TO_PARSE_UINT = -6,
-    CASPER_FAILED_TO_GET_RESPONSE = -7,
-    CASPER_FAILED_TO_PARSE_RESPONSE = -8,
-    CASPER_FILE_ALREADY_EXISTS = -9,
-    CASPER_UNSUPPORTED_ALGORITHM = -10,
-    CASPER_REPSONSE_IS_ERROR = -11,
-    CASPER_INVALID_JSON = -12,
-    CASPER_INVALID_RPC_RESPONSE = -13,
-    CASPER_FAILED_SENDING = -14,
-    CASPER_IO_ERROR = -15,
-    CASPER_TO_BYTES_ERROR = -16,
-    CASPER_CRYPTO_ERROR = -17,
-    CASPER_INVALID_CL_VALUE = -18,
-    CASPER_INVALID_ARGUMENT = -19,
-    CASPER_INVALID_RESPONSE = -20,
-    CASPER_FFI_SETUP_NOT_CALLED = -21,
-    CASPER_FFI_PTR_NULL_BUT_REQUIRED = -22,
-    CASPER_CONFLICTING_ARGUMENTS = -23,
+    CASPER_DEPLOY_SIZE_TOO_LARGE = -7,
```

Comment:
Wouldn't it be better to make this backwards compatible and add the new error type at the end, rather than changing all of the exit codes that come after this?

Reply:
agreed